### PR TITLE
📝 Add a GitHub-specific security page

### PR DESCRIPTION
**Pull request by webknjaz**
It will show up @ https://github.com/python/cpython/security/policy allowing to navigate users who get there from "Security" tab in the GitHub repo to the full article explaining the security vulnerability reporting practices.

### CODE REVIEW

File: .github/SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Security Policy
+
+## Supported Versions
+
+Python team applies security fixes according to the table maintained as
```
Comment (hugovk) [inline .github/SECURITY.md]:
```suggestion
The Python team applies security fixes according to the table in
```

File: .github/SECURITY.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+a part of [the devguide](
```
Comment (hugovk) [inline .github/SECURITY.md]:
```suggestion
in [the devguide](
```

File: .github/SECURITY.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+https://devguide.python.org/#status-of-python-branches
+). Please check it out for more details.
```
Comment (hugovk) [inline .github/SECURITY.md]:
```suggestion
).
```

File: .github/SECURITY.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+
+## Reporting a Vulnerability
+
+Please check out the guidelines on reporting security issues [on the
```
Comment (hugovk) [inline .github/SECURITY.md]:
```suggestion
Please read the guidelines on reporting security issues [on the
```

File: .github/SECURITY.md [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+official website](
+https://www.python.org/news/security/#reporting-security-issues-in-python
+) for the instructions on how to report a security-related problem to
```
Comment (hugovk) [inline .github/SECURITY.md]:
```suggestion
) for instructions on how to report a security-related problem to
```

File: .github/SECURITY.md [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+the Python team responsibly.
+
+To reach the response team, send email to `security at python dot org`.
```
Comment (hugovk) [inline .github/SECURITY.md]:
```suggestion
To reach the response team, email `security at python dot org`. 
```

**Review by hugovk — COMMENTED** [review]
Minor edits for brevity.

**Comment by asvetlov** [comment]
I've merged the PR to stop people creating similar duplicating pull requests.
The PR is good enough as a starter at least.
If we need to improve the text let's do it in a separate PR.

**Comment by asvetlov** [comment]
Automerge bot is slow, did it myself

**Comment by webknjaz** [comment]
FTR here's the resulting view: https://github.com/python/cpython/security/policy

**Pull request merged**